L/librealsense/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "librealsense"
+version = v"2.31.0"
+
+# Collection of sources required to build librealsense
+sources = [
+    "https://github.com/IntelRealSense/librealsense.git" =>
+    "bdce5a4ebbd716e9d7372ad8831a929bccccd0b2",
+
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd librealsense/
+mkdir build
+cd build
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_EXAMPLES=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_GRAPHICAL_EXAMPLES=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_UNIT_TESTS=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_WITH_OPENMP=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DENFORCE_METADATA=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_PYTHON_BINDINGS=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_MATLAB_BINDINGS=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_OPENNI2_BINDINGS=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_NODEJS_BINDINGS=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DBUILD_CSHARP_BINDINGS=false"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DENABLE_CCACHE=false"
```

Comment:
Any particular reason for disabling ccache?  It makes rebuilds much faster

Reply:
I didn't set `BINARYBUILDER_USE_CCACHE` on my machine. I thought this CMake flag would take no effect, but yes, it should be turned on when building on Azure.